src/pages/ThorChainLP/components/AddLiquidity/AddLiquidityInput.tsx

```diff
@@ -1424,10 +1423,11 @@ export const AddLiquidityInput: React.FC<AddLiquidityInputProps> = ({
             !hasEnoughAssetBalance ||
             !hasEnoughRuneBalance ||
             isApprovalTxPending ||
-            isSweepNeededLoading ||
+            isSweepNeeded === undefined ||
```

Comment:
Looking at the `enabled` args this looks like it would fail for RUNE Txs

Reply:
good catch. fixed here: https://github.com/shapeshift/web/pull/6419/commits/aff905fc57d6d98035f538d8eefa0630368ffc0b